manim/mobject/geometry/labeled.py

```diff
@@ -0,0 +1,154 @@
+r"""Mobjects that inherit from lines and contain a label along the length."""
+
+from __future__ import annotations
+
+__all__ = ["LabeledLine", "LabeledArrow"]
+
+from manim.constants import *
+from manim.mobject.geometry.line import Arrow, Line
+from manim.mobject.geometry.shape_matchers import (
+    BackgroundRectangle,
+    SurroundingRectangle,
+)
+from manim.mobject.text.tex_mobject import MathTex, Tex
+from manim.mobject.text.text_mobject import Text
+from manim.utils.color import *
```

Comment:
This is only imported for the constant WHITE, which could be removed since a default color of None makes it WHITE anyways.
```suggestion
```